tests/test_fdb_update.py

```diff
@@ -0,0 +1,119 @@
+    vlan_before = how_many_entries_exist(dvs.adb, "ASIC_STATE:SAI_OBJECT_TYPE_VLAN")
+    bp_before = how_many_entries_exist(dvs.adb, "ASIC_STATE:SAI_OBJECT_TYPE_BRIDGE_PORT")
+    vm_before = how_many_entries_exist(dvs.adb, "ASIC_STATE:SAI_OBJECT_TYPE_VLAN_MEMBER")
+
+    # create vlan
+    create_entry_tbl(
+        dvs.cdb,
+        "VLAN", "Vlan2",
+        [
+            ("vlanid", "2"),
+        ]
+    )
+
+    # create vlan member entry in application db
+    create_entry_tbl(
+        dvs.cdb,
+        "VLAN_MEMBER", "Vlan2|Ethernet0",
+        [
+            ("tagging_mode", "untagged"),
+        ]
+    )
+
+    # check that the vlan information was propagated
+    vlan_after = how_many_entries_exist(dvs.adb, "ASIC_STATE:SAI_OBJECT_TYPE_VLAN")
+    bp_after = how_many_entries_exist(dvs.adb, "ASIC_STATE:SAI_OBJECT_TYPE_BRIDGE_PORT")
+    vm_after = how_many_entries_exist(dvs.adb, "ASIC_STATE:SAI_OBJECT_TYPE_VLAN_MEMBER")
+
+    assert vlan_after - vlan_before == 1, "The Vlan2 wasn't created"
+    assert bp_after - bp_before == 1, "The bridge port wasn't created"
+    assert vm_after - vm_before == 1, "The vlan member wasn't added"
+
+    # Get mapping between interface name and its bridge port_id
+    iface_2_bridge_port_id = dvs.get_map_iface_bridge_port_id(dvs.adb)
+
+    # check that the FDB entry was inserted into ASIC DB
+    assert how_many_entries_exist(dvs.adb, "ASIC_STATE:SAI_OBJECT_TYPE_FDB_ENTRY") == 1, "The fdb entry wasn't inserted to ASIC"
+
+    ok, extra = dvs.is_fdb_entry_exists(dvs.adb, "ASIC_STATE:SAI_OBJECT_TYPE_FDB_ENTRY",
+                    [("mac", "52-54-00-25-06-E9"), ("vlan", "2")],
+                    [("SAI_FDB_ENTRY_ATTR_TYPE", "SAI_FDB_ENTRY_TYPE_DYNAMIC"),
+                     ("SAI_FDB_ENTRY_ATTR_BRIDGE_PORT_ID", iface_2_bridge_port_id["Ethernet0"]),
+                     ('SAI_FDB_ENTRY_ATTR_PACKET_ACTION', 'SAI_PACKET_ACTION_FORWARD')]
+    )
+    assert ok, str(extra)
+
+    # create vlan member entry in application db
+    create_entry_tbl(
+        dvs.cdb,
+        "VLAN_MEMBER", "Vlan2|Ethernet4",
+        [
+            ("tagging_mode", "untagged"),
+        ]
+    )
+
+    # update FDB entry port in Application DB
+    create_entry_pst(
+        dvs.pdb,
+        "FDB_TABLE", "Vlan2:52-54-00-25-06-E9",
+        [
+            ("port", "Ethernet4"),
+            ("type", "dynamic"),
+        ]
+    )
+
+    # Get mapping between interface name and its bridge port_id
+    iface_2_bridge_port_id = dvs.get_map_iface_bridge_port_id(dvs.adb)
+    
+    ok, extra = dvs.is_fdb_entry_exists(dvs.adb, "ASIC_STATE:SAI_OBJECT_TYPE_FDB_ENTRY",
+                    [("mac", "52-54-00-25-06-E9"), ("vlan", "2")],
+                    [("SAI_FDB_ENTRY_ATTR_TYPE", "SAI_FDB_ENTRY_TYPE_DYNAMIC"),
+                     ("SAI_FDB_ENTRY_ATTR_BRIDGE_PORT_ID", iface_2_bridge_port_id["Ethernet4"]),
+                     ('SAI_FDB_ENTRY_ATTR_PACKET_ACTION', 'SAI_PACKET_ACTION_FORWARD')]
+    )
+    assert ok, str(extra)
```

Comment:
could you tear down all the configurations after the test is finished?

Reply:
yes, we have add it. Pls review